pyramid_nacl_session/session.py

```diff
@@ -1,15 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please do not add this to Python files in this repository. The default is UTF-8 on Python 3 anyway.